js/package.json

```
@@ -4,6 +4,10 @@
     "terser": "5.37.0"
   },
   "devDependencies": {
-    "@happy-dom/global-registrator": "latest"
+    "@happy-dom/global-registrator": "latest",
```

Biome (1.9.4)

[error] The key @happy-dom/global-registrator was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)